include/net-snmp/library/parse.h

```diff
@@ -234,16 +234,10 @@ SOFTWARE.
     void            snmp_mib_toggle_options_usage(const char *lead,
                                                   FILE * outf);
     NETSNMP_IMPORT
-    void            print_mib(FILE *);
-    NETSNMP_IMPORT
```

Comment:
Please keep this declaration and remove the declaration from include/net-snmp/mib_api.h instead.

Reply:
Done.